routes/seaice.py

```diff
@@ -92,3 +101,25 @@ def run_point_fetch_all_seaice(lat, lon):
         if hasattr(exc, "status") and exc.status == 404:
             return render_template("404/no_data.html"), 404
         return render_template("500/server_error.html"), 500
+
+
+@routes.route("/seaice/enddate/")
+def seaice_enddate():
+    """Get the latest date of sea ice concentration data available in the Rasdaman server.
+    Args:
+        None
+
+    Returns:
+        JSON-like dict of the latest year and month of sea ice concentration data
+
+    """
+    hsia_encodings = asyncio.run(get_seaice_metadata())
+    latest_date = hsia_encodings["ansi"][-1]
+
+    try:
+        valid_date = validate_seaice_timestring(latest_date)
+        return {"year": valid_date.year, "month": valid_date.month}
+    except ValueError as e:
+        print(e)
```

Comment:
Remove the print (at least, issue it to the logger).

I feel like this should throw a 503 error, not return a data stub.